src/components/SubPage/SubPage.css

```diff
@@ -1,7 +1,7 @@
 .SubPage {
   margin: calc(var(--s__unit) * 8) 0;
 
-  &.has-image {
+  &__with-image {
```

Comment:
shouldn't this be a modifier?

Reply:
According to the current logic `SubPage` component could have an image which triggers its custom className - `SubPage__with-image`.
If there's image alignment modifier (prop) we're conditionally adding a modifier `SubPage__with-image--left-aligned` or `SubPage__with-image--right-aligned`.

Reply:
Your explanation further proves my point -- that's a modifier, not an element inside a block.

Reply:
Now I got the point! Fixed this issue.